tests/log-data-per-sec-interval-10/test.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
+  min-version: 7.0.2
+  features:
+    - DEBUG
+  files:
+    - src/counters.c
+    - src/decode.c
+
+pcap: input.pcap
```

Comment:
This is used in case we want to indicate that the sv test doesn't require a pcap, or to inform Suricata-verify what's the pcap path - only if the test directory doesn't contain a pcap file.

This last case is what we must do if the test uses an existing pcap - we do that to avoid file duplication.

That explained, I've noticed that at least two of your tests have the same pcap. Could you please remove the pcap duplication for the other tests from this contribution? You can follow [this example](https://github.com/OISF/suricata-verify/blob/master/tests/base64-issue-5223-6/test.yaml#L1) for how to refer to a pcap from a different test.